server/plugin/api.go

```diff
@@ -369,11 +370,7 @@ func (p *Plugin) testAPI(w http.ResponseWriter, r *http.Request) {
 		w.Write([]byte(err.Error()))
 		return
 	}
-	// TODO: for testing purposes, remove later
-	mattermostUserID := r.Header.Get(constants.HeaderMattermostUserIDAPI)
-	u, _ := p.Store.LoadUser(mattermostUserID)
-	t, _ := p.ParseAuthToken(u.AccessToken)
-	fmt.Println("\n\n\n", t)
+
 	res, _ := json.Marshal(response)
```

Comment:
Why not checking error?

Reply:
removed this test API from the next PR so you can ignore it for now